community/modules/scheduler/htcondor-configure/README.md

```diff
@@ -76,12 +76,26 @@ example can be found in the [examples README][htc-example].
 
 This module supports high availability modes of the HTCondor Central Manager and
 of the Access Points. In these modes, the services can be resiliant against
-zonal failures by distributing the services across two zones. To implement
-HA of the Central Manager service, apply the startup script in the example
-snipppet to 2 VMs. The 2 VMs should differ only by using the primary and
-secondary IP addresses created by this module.
+zonal failures by distributing the services across two zones. Modify the above
+example by setting `central_manager_high_availability` to `true` and adding a
+new deployment variable `zone_secondary` set to another zone in the same region.
+The 2 VMs can use the same startup script, but should differ by setting:
+
+- primary and secondary zones defined in deployment variables
+- primary and secondary IP addresses created by this module
+- differing name prefixes
 
 ```yaml
+# vars:
+#   zone_secondary: a zone in same region as vars.zone
```

Comment:
nit: What is the value of commenting these lines out? I would maybe suggest something like:
```
vars:
...
  zone_secondary: a zone in same region as vars.zone
```

Reply:
Made change based on offline discussion.